docs/Using-Fleet/Vulnerability-Processing.md

```diff
@@ -200,4 +201,46 @@ If there's a match, we remove the false positive from the list.
 
 - CentOS 6, 7 and 8.
 
+## What Fleet checks for vulnerabilites
+
+Currently, Fleet detects vulnerabilities for installed software that falls into the following categories (types):
+
+### Apps
+- macOS
+- Windows
+
+### Browser plugins
+- macOS, Windows, Linux
+  - Chrome extensions
+  - Firefox extensions
+
+### Packages
+- macOS
+  - Python
+  - All software installed using Homebrew
+
+- Windows
+  - Python
+  - Atom
+  - All software installed using Chocolatey
+
+- Linux
+  - deb
+  - RPM
+  - Atom
+  - Python
+  - All software installed using APT (Debian)
+  - All software installed using Portage (Gentoo)
+  - All software installed using YUM (Red Hat)
+  - All software installed using NPM
+
+Fleet collects this software by running a software collection osquery query on all hosts.
```

Comment:
```suggestion
Fleet collects this software by running one of the following osquery queries on all hosts.
```

I suggest shortening this sentence for clarity. 

Also, it seems that by adding this section we may be able to de-duplicate the following section of the same page: https://github.com/fleetdm/fleet/pull/5435/files#diff-646de0b00b525c7087eaae2cc821c3b65e8747ea692c450e11b8b2fcf42215d4R126-R130

I think, in the section I link to above, we can replace the list of queries with a link to this new section. 

This means, the section I link to above becomes something like "The ingestion of software varies per platform. For each platform we run an [osquery query](link to your new section) to ingest software."